docs/source/index.rst

```diff
@@ -53,7 +53,7 @@ Topics in this section will help you get started with ExecuTorch.
      .. grid-item-card:: :octicon:`file-code;1em`
         Getting started with ExecuTorch
         :img-top: _static/img/card-background.svg
-        :link: getting-started-setup.html
+        :link: getting-started.html
```

Comment:
Can you change the "Getting Started" page and links to the new export page? 

In the "Next Steps" section, as well as "Preparing the Model" section.